doctr/models/_utils.py

```diff
@@ -161,3 +161,18 @@ def get_language(text: str) -> Tuple[str, float]:
     if len(text) <= 1 or (len(text) <= 5 and lang.prob <= 0.2):
         return "unknown", 0.0
     return lang.lang, lang.prob
+
+
+def invert_between_dict_of_lists_and_list_of_dicts(
```

Comment:
`invert_data_structure` with a clear docstring which explains the typing  maybe ? so that we keep a short and clear name ?

Reply:
Sounds good :)

Reply:
Done